utf8/test.out.txt

```diff
@@ -0,0 +1 @@
+�􏿿􏿿􏿿􏿿􏿿􏿿􏿿􏿿
```

Comment:
I was trying to find where this file is referenced. Is this possible some test data not intended for including?